packages/codefixes/src/codefixes.ts

```diff
@@ -6,13 +6,18 @@ import { AddErrorTypeGuardCodeFix } from './fixes/addErrorTypeGuard/index.js';
 import { AddMissingExportCodeFix } from './fixes/addMissingExport/index.js';
 import { MakeMemberOptionalCodeFix } from './fixes/makeMemberOptional/index.js';
 import { AddMissingTypesBasedOnInheritanceCodeFix } from './fixes/addMissingTypesBasedOnInheritance/index.js';
+import { AddMissingTypesBasedOnInlayHintsCodeFix } from './fixes/addMissingTypesBasedOnInlayHints/index.js';
 
 export const codefixes = new CodeFixesProvider([
+  new BaseCodeFixCollection({
+    7050: new AddMissingTypesBasedOnInheritanceCodeFix(),
```

Comment:
Need to be called before `7050: new AddMissingTypesBasedOnInlayHintsCodeFix(),` but can't place it to the following array because of duplicate keys. 

Thinking about refactor collection to  accept:

```
7050: [
  new AddMissingTypesBasedOnInheritanceCodeFix(),
  new AddMissingTypesBasedOnInlayHintsCodeFix(),
]
```

and few other ways... but don't want to do it in this PR.